channeldb/channel.go

```diff
@@ -1440,6 +1445,38 @@ func (c *OpenChannel) UpdateCommitment(newCommitment *ChannelCommitment,
 				"updates: %v", err)
 		}
 
+		// Persist the peer unsigned acked updates that are not included
+		// in our new commitment.
+		updateBytes := chanBucket.Get(peerUnsignedAckedUpdatesKey)
+		if updateBytes == nil {
+			return nil
+		}
+
+		r := bytes.NewReader(updateBytes)
+		updates, err := deserializeLogUpdates(r)
+
+		var validUpdates []LogUpdate
+
+		for _, upd := range updates {
+			lIdx := upd.LogIndex
+
+			// Check if it's not on our local commitment
+			if lIdx >= newCommitment.LocalLogIndex {
```

Comment:
No need for the intermediate variable here? (assuming if fits within the char limit)